### PR TITLE
New feature: Fate Count Calculator

### PR DESCRIPTION
**Pull request by HansonSin**
Hi! This is my first time trying to contribute to an open-source project. I tried to implement the idea in issue #143 and this is how it looks like.

https://user-images.githubusercontent.com/46597093/149138006-80e47a8e-f2d4-4868-9e06-845bbdc6a7c1.mp4

### CODE REVIEW

**Comment by MadeBaruna** [comment]
Thanks for your contribution, will check it on sunday

File: src/routes/calculator/_fateCount.svelte [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+              <tr>
+                <th class="text-white pr-2 text-center">{$t('calculator.fateCount.type')}</th>
+                <th class="text-white pr-2 text-center">{$t('calculator.fateCount.amount')}</th>
+                <th class="text-white pr-2 text-right">{$t('calculator.fateCount.totalPrimogem')}</th>
+              </tr>
+              {#each result as res}
+                <tr>
+                  <td class="border-t border-gray-700 text-white pr-2">
+                    <span class="inline-flex align-middle">
+                      {#if res.image !== null}<img src={res.image} alt={res.name} class="w-6 mr-2" />{/if}
```
Comment (MadeBaruna) [inline src/routes/calculator/_fateCount.svelte:128]: image
limit the icon height

File: src/routes/calculator/_fateCount.svelte [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+          </div>
+        </div>
+      {/each}
+    </div>
+    <div class="md:col-span-2 xl:col-span-1">
+      <Button className="block w-full md:w-auto" on:click={calculate}
+        >{$t('calculator.fateCount.calculate')}</Button
+      >
+      {#if result !== null}
+        <script>console.log(result);</script>
```
Comment (MadeBaruna) [inline src/routes/calculator/_fateCount.svelte:111]: remove please

**Review by MadeBaruna — CHANGES_REQUESTED** [review]

File: src/routes/calculator/_fateCount.svelte (hunk 2 above)
Comment (HansonSin) [reply src/routes/calculator/_fateCount.svelte]: oops sorry

File: src/routes/calculator/_fateCount.svelte (hunk 1 above)
Comment (HansonSin) [reply src/routes/calculator/_fateCount.svelte]: done

**Pull request merged**